blog/app/models/micropost.rb

```diff
@@ -0,0 +1,26 @@
+class Micropost < ApplicationRecord
+  MICROPOST_PARAMS = %i(content image).freeze
+
+  belongs_to :user
+
+  has_one_attached :image
+
+  delegate :name, to: :user
```

Comment:
2 dòng này có tác dụng gì nhỉ

Reply:
- has_one_attached: Cụ thể hóa mối quan hệ giữa 1 file và model (Active Storage sẽ đảm bảo việc nối giữa các các bản ghi của mình với 1 file đính kèm )
- delegate: giúp dễ dàng gọi các public methods của object khác giống như là của chính mình , Đặc biệt với các models có quan hệ phức tạp thì bạn có thể thực hiện gọi từ một class đến nhiều class khác.
- delegate :name, to: :user có tác dụng là giúp chúng ta lấy tên người viết bài đơn giản hơn thay vì viết micropost.user.name thì ta có thể viết micropost.name là lấy được tên người viết.